### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by Mange99**

### CODE REVIEW

File: fdsfds [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  [31morigin/Deaa1[m
+  [31morigin/Deaa2[m
+  [31morigin/DeaaFilter[m
+  [31morigin/FavoritesBranch[m
+  [31morigin/HEAD[m -> origin/main
+  [31morigin/Magnus_knapp[m
+  [31morigin/TbButton[m
+  [31morigin/TipsPage-Sara's[m
+  [31morigin/body_specific_excercise[m
+  [31morigin/calculators[m
```
Comment (Torbi) [inline fdsfds:13]: wtf is this

File: ui/src/components/exerciseCardGrid/exerciseCardGrid.tsx [hunk 2]
```diff
@@ -7,9 +7,17 @@ export function ExerciseCardGrid({
   exercises,
 }: ExerciseCardGridProps) {
   return (
-    <Grid templateColumns="repeat(3, 1fr)" rowGap={8} columnGap={8}>
+    <Grid
+      templateColumns={{
+        sm: "repeat(1, 1fr)",
+        lg: "repeat(2, 1fr)",
+        xl: "repeat(3, 1fr)",
+      }}
```
Comment (Torbi) [inline ui/src/components/exerciseCardGrid/exerciseCardGrid.tsx:15]: Delete component?

File: ui/src/components/modalInfoPanel/infoPanel.tsx [hunk 3]
```diff
@@ -15,18 +16,21 @@ const InfoPanel: React.FC<InfoPanelProps> = ({ show, onHide, exercise }) => {
...
-          dapibus ac facilisis in, egestas eget quam. Morbi leo risus, porta ac
-          consectetur ac, vestibulum at eros.
-        </p>
+        <Text fontSize={"xl"} fontWeight="medium">
+          This exercise trains the {exercise.bodyPart}, more specific{" "}
+          {exercise.target}.
+        </Text>
+        <Text>
+          You need {exercise.equipment} to do this exercise. It is demonstrated
+          here below in a short video.
```
Comment (Torbi) [inline ui/src/components/modalInfoPanel/infoPanel.tsx]: gif instead of video?

File: ui/src/pages/LandingPage.tsx [hunk 4]
```diff
@@ -2,10 +2,12 @@ import {
   Box,
   Button,
   Container,
+  Flex,
   Grid,
   GridItem,
```
Comment (Torbi) [inline ui/src/pages/LandingPage.tsx]: not used

File: ui/src/pages/profilePage.tsx [hunk 5]
```diff
@@ -27,76 +28,82 @@ export const ProfilePage = () => {
...
+        >
+          {session != [] &&
+            session.map((session) => {
+              return (
+                <Box
+                  margin="auto"
+                  w="80%"
+                  p={4}
+                  borderRadius="2xl"
+                  boxShadow={"base"}
```
Comment (Torbi) [inline ui/src/pages/profilePage.tsx:77]: have you rewritten everything in chakra?

**Review by Torbi — CHANGES_REQUESTED** [review]

**Comment by Torbi** [comment]
ExercisePage is very big, cant break it up in components?

File: ui/src/components/exerciseCardGrid/exerciseCardGrid.tsx (hunk 2 above)
Comment (Mange99) [reply ui/src/components/exerciseCardGrid/exerciseCardGrid.tsx:15]: good example of a bad idea

File: ui/src/pages/profilePage.tsx (hunk 5 above)
Comment (Mange99) [reply ui/src/pages/profilePage.tsx:77]: The code in question is written in chakra ui

**Pull request merged**